### DISCUSSION TITLE
Using in mono repo

### DISCUSSION THREAD

**Discussion by bpietravalle** [gethyas/hyas#212] [Q&A] [answered]
Hi there,  any suggestion on how to apply this template in a mono repo?  My repo is structured:
```
src/site1
src/site2
themes/theme1
themes/theme2
```
most sites use more than one theme.

I'm having trouble getting the css post processing to work.  Should each `src/site` folder have it's own `postcss.config.js`?

Thanks for your help,
Brian

**Comment by h-enk** [comment, marked as answer]
As far as I know/tried, it's not possible to use Hyas in the theme directory (mainly because of the npm setup).

However, you could have multiple separate instances of Hyas in a single repo (each with it's own `postcss.config.js`).

**Reply by bpietravalle** [reply]
thanks for the suggestion.  any idea what's causing the issue with the npm setup?  I'll try putting `postcss.config.js` and babel in `src/site` folders. hopefully, postcss will find the right files in `themes/theme{n}/layouts` 🤞

**Reply by h-enk** [reply]
As I recall it, it's because you would then not be able to run the npm scripts located in the themes subdirectory (without adaptations in the project root directory).
